.github/workflows/optional-test.yaml

```diff
@@ -33,6 +33,8 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
+        with:
+          cache: 'yarn'
```

Comment:
Is this lacking an install deps step?

Reply:
Apparently no yarn install is needed here because it has no dependencies, so cache is not needed as well